util.go

```diff
@@ -47,9 +49,38 @@ func mustGetFlagDuration(cmd *cobra.Command, name string) time.Duration {
 	return val
 }
 
+func parseAddr(addr string) (string, error) {
```

Comment:
Nit: seems like this stuff isn't really related to the other stuff (namely the `dev test pkg/...` thing), so you could put it in its own commit.